managed/ui/src/components/config/Security/KeyManagementConfigurationContainer.js

```diff
@@ -32,26 +33,35 @@ const mapDispatchToProps = (dispatch) => {
     fetchKMSConfigList: () => {
       return dispatch(fetchAuthConfigList()).then((response) =>
         dispatch(fetchAuthConfigListResponse(response.payload))
-      );
+      )
+      .catch((err) => {
+        toast.error("Error occured while fetching config.")
+      })
     },
 
     setKMSConfig: (provider, body) => {
       return dispatch(createKMSProviderConfig(provider, body))
         .then((response) => {
-          return dispatch(createKMSProviderConfigResponse(response.payload));
+          return dispatch(createKMSProviderConfigResponse(response.payload)).then(
+            response => {debugger
+               toast.success("Successfully added the configuration") }
```

Comment:
same here - change to `'` and add `;` pls

Reply:
Done